### PR TITLE
improve test coverage for branchings.py

### LINKED ISSUE #6520: Improve test coverage for branchings.py
The [coverage on branchings.py](https://app.codecov.io/gh/networkx/networkx/blob/main/networkx/algorithms/tree/branchings.py) is 92.37 and I want to work on it to improve it.

### PR DESCRIPTION
**Pull request by Qudirah**
Fixes #6520 
@MridulS @rossbar @paulitapb Please have a look

### CODE REVIEW

**Review by mjschwenne — APPROVED** [review]

File: networkx/algorithms/tree/tests/test_branchings.py [hunk 1]
```diff
@@ -199,6 +200,26 @@ def test_greedy_max1():
     assert_equal_branchings(B, B_)
 
 
+def test_greedy_branch_kind():
+    G = G1()
+    with pytest.raises(nx.NetworkXException, match="Unknown value for `kind`."):
+        B = branchings.greedy_branching(G, kind="lol")
+
+
+def test_edges():
```
Comment (dschult) [inline networkx/algorithms/tree/tests/test_branchings.py:209]: Is this a fair description of the test?
```suggestion
def test_greedy_branching_for_unsortable_nodes():
```

File: networkx/algorithms/tree/tests/test_branchings.py [hunk 2]
```diff
@@ -199,6 +200,26 @@ def test_greedy_max1():
     assert_equal_branchings(B, B_)
 
 
+def test_greedy_branch_kind():
```
Comment (dschult) [inline networkx/algorithms/tree/tests/test_branchings.py:203]:
```suggestion
def test_greedy_branching_kwarg_kind():
```

**Review by dschult — COMMENTED** [review]
Looks good!  
I have two small suggestions about names of the tests.

**Comment by Qudirah** [comment]
@dschult  Thank you. I agree.

**Pull request merged**